src/main/java/org/openrewrite/java/logging/slf4j/JulParameterizedArguments.java

```diff
@@ -101,18 +152,18 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, Execu
                     return method;
                 }
                 String newName = getMethodIdentifier(levelArgument);
-                if(newName == null) {
+                if (newName == null) {
                     return method;
                 }
                 maybeRemoveImport("java.util.logging.Level");
 
                 String originalFormatString = Objects.requireNonNull((String) ((J.Literal) messageArgument).getValue());
                 List<Integer> originalIndices = originalLoggedArgumentIndices(originalFormatString);
-                List<Expression> originalParameters = originalParameters(originalArguments.get(2));
+                List<Expression> stringFormatArguments = splitArrayToExpressions(originalArguments.get(2), originalIndices.size());
 
                 List<Expression> targetArguments = new ArrayList<>(2);
                 targetArguments.add(buildStringLiteral(originalFormatString.replaceAll("\\{\\d*}", "{}")));
-                originalIndices.forEach(i -> targetArguments.add(originalParameters.get(i)));
+                originalIndices.forEach(i -> targetArguments.add(stringFormatArguments.get(i)));
                 return JavaTemplate.builder(createTemplateString(newName, targetArguments))
                         .contextSensitive()
```

Comment:
```suggestion
```

I don't think you need it here (without it is better, as the `contextSensitive` prop disables caching).